packages/flutter/lib/src/cupertino/theme.dart

```diff
@@ -0,0 +1,391 @@
+  final Brightness _brightness;
+  /// The general brightness theme of the [CupertinoThemeData].
+  ///
+  /// Affects all other theme properties when unspecified. Defaults to
+  /// [Brightness.light].
+  ///
+  /// If coming from a Material [Theme] and unspecified, [brightness] will be
+  /// derived from the Material [ThemeData]'s `brightness`.
+  ///
+  /// Reading this property from [CupertinoTheme.of] will create a dependency
+  /// from the [BuildContext] to changes in this property.
+  Brightness get brightness => _brightness ?? Brightness.light;
+
+  final Color _primaryColor;
+  /// A color used on interactive elements of the theme.
+  ///
+  /// This color is generally used on text and icons in buttons and tappable
+  /// elements. Defaults to blue or orange when [brightness] is light or dark.
+  ///
+  /// If coming from a Material [Theme] and unspecified, [primaryColor] will be
+  /// derived from the Material [ThemeData]'s `colorScheme.primary`. However, in
+  /// iOS styling, the [primaryColor] is more sparsely used than in Material
+  /// Design where the [primaryColor] can appear on non-interactive surfaces like
+  /// the [AppBar] background, [TextField] borders etc.
+  ///
+  /// Reading this property from [CupertinoTheme.of] will create a dependency
+  /// from the [BuildContext] to changes in this property.
+  Color get primaryColor {
+    return _primaryColor ??
+        (_isLight ? CupertinoColors.activeBlue : CupertinoColors.activeOrange);
+  }
+
+  final Color _primaryContrastingColor;
+  /// A color used on content that contrast against a [primaryColor] background.
+  ///
+  /// This color is used on text and icons against a filled [CupertinoButton]
+  /// whose background is the [primaryColor] for instance.
+  ///
+  /// If coming from a Material [Theme] and unspecified, [primaryContrastingColor]
+  /// will be derived from the Material [ThemeData]'s `colorScheme.onPrimary`.
+  ///
+  /// Reading this property from [CupertinoTheme.of] will create a dependency
+  /// from the [BuildContext] to changes in this property.
+  Color get primaryContrastingColor {
+    return _primaryContrastingColor ??
+        (_isLight ? CupertinoColors.white : CupertinoColors.black);
+  }
+
+  final CupertinoTextTheme _textTheme;
+  /// Text styles used by Cupertino widgets.
+  ///
+  /// Derived from [brightness] and [primaryColor] if unspecified, including
+  /// [brightness] and [primaryColor] of a Material [ThemeData] if coming
+  /// from a Material [Theme].
+  ///
+  /// Reading this property from [CupertinoTheme.of] will create a dependency
+  /// from the [BuildContext] to changes in this property.
+  CupertinoTextTheme get textTheme {
+    return _textTheme ?? CupertinoTextTheme(
+      isLight: _isLight,
+      primaryColor: primaryColor,
+    );
+  }
+
+  final Color _barBackgroundColor;
+  /// Background color of the top nav bar and bottom tab bar.
+  ///
+  /// Defaults to a light gray or a dark gray translucid color depending
```

Comment:
translucent

Best to refer to the actual color constants.

Reply:
done